test/suite/test_backup01.py

```diff
@@ -31,15 +31,15 @@
 import shutil
 import string
 import time
-from suite_subprocess import suite_subprocess
-import wiredtiger, wttest
+from test_backup_base import test_backup_base
```

Comment:
In the `test/suite` directory, only actual tests have a `test_` prefix to the filename. I think this should be in a file perhaps called `wtbackup.py` and functions named appropriately. I suggest looking at `wtdataset.py` for one example.

Reply:
awesome, changed it to wtbackup.py